specification/cognitiveservices/data-plane/ComputerVision/v1.0/ComputerVision.json

```diff
@@ -0,0 +1,1482 @@
+                }
+            }
+        },
+        "/describe?overload=url": {
+            "post": {
+                "description": "This operation generates a description of an image in human readable language with complete sentences.  The description is based on a collection of content tags, which are also returned by the operation. More than one description can be generated for each image.  Descriptions are ordered by their confidence score. All descriptions are in English. Two input methods are supported -- (1) Uploading an image or (2) specifying an image URL.A successful response will be returned in JSON.  If the request failed, the response will contain an error code and a message to help understand what went wrong.",
+                "operationId": "DescribeImage",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "maxCandidates",
+                        "in": "query",
+                        "description": "Maximum number of candidate descriptions to be returned.  The default is 1.",
+                        "type": "string",
+                        "required": false,
+                        "default": "1"
+                    },
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/SubscriptionKey"
+                    },
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/ImageUrl"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "schema": {
+                            "$ref": "#/definitions/ImageDescription"
+                        }
+                    },
+                    "default": {
+                        "schema": {
+                            "$ref": "#/definitions/ComputerVisionError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Successful Describe request": {
+                        "$ref": "./examples//SuccessfulDescribeWithUrl.json"
+                    }
+                }
+            }
+        },
+        "/tag?overload=url": {
+            "post": {
+                "description": "This operation generates a list of words, or tags, that are relevant to the content of the supplied image. The Computer Vision API can return tags based on objects, living beings, scenery or actions found in images. Unlike categories, tags are not organized according to a hierarchical classification system, but correspond to image content. Tags may contain hints to avoid ambiguity or provide context, for example the tag “cello” may be accompanied by the hint “musical instrument”. All tags are in English.",
+                "operationId": "TagImage",
```

Comment:
`operationId` must be `groupname_methodname` -- 

`groupname` is used to create a class in most generated languages
`methodname` is used to generate a method name in most languages